k8s/k8s-aws-efs-csi-driver/_variables.tf

```diff
@@ -0,0 +1,29 @@
+variable "md_metadata" {
```

Comment:
Let’s just pass the name prefix in here instead of the md_metadata object so this can be MD agnostic

Reply:
Instead of "name_prefix" literally, I've noticed we're using "name" in these modules. For context

Reply:
"these" => the modules used in app templates. See comment above.

Reply:
Done